### DISCUSSION TITLE
Missing app in app-store?

### DISCUSSION THREAD

**Discussion by innuendoz** [runtipi/runtipi-appstore#4408] [Q&A] [answered]
hi,
I have runtipi running at home and on a PC at work.  I noticed there's an app by the name of Wallos that is available in the app-store on one PC but not the other.  I'm able to deploy it manually with docker and both PCs have the latest version of runtipi, but it just isn't listed.  Not a big deal, I just noticed the one app and was curious is there may be more.

**Comment by cori** [comment]
If you're running two separate instances of Tipi, does either of them have a value in the `Apps repo URL` setting on the Settings > Settings page?

<img width="414" alt="image" src="https://github.com/user-attachments/assets/63243159-7a58-4763-a55d-9a861c328297">

<img width="584" alt="image" src="https://github.com/user-attachments/assets/9b704772-4d31-44a1-8834-359768d1f2a2">

**Reply by innuendoz** [reply]
they both have the same url set, https://github.com/runtipi/runtipi-appstore

**Comment by cori** [comment, marked as answer]
Huh i could have sworn i saw a reference to architecture in this discussion, but i guess I was conflating it with something else (maybe in Discord?).

Anyway I have a custom app store running at https://github.com/cori/runtipi-appstore, and i've added the "other" architecture to Wallos's config in my store. You could try connecting Tipi on the machine that's missing Wallos to my app store to see if it shows up, in which case my baseline assumption is that it works fine on an arm processor but the Tipi app author didn't include that for some reason.

If you're mistrustful of running code from someone else's app store on your device (and you should be) [here's a link to the diff](https://github.com/cori/runtipi-appstore/commit/b4ff28ad64c9e9d0d55c4a9cf4656895db6b659b) so you can verify the change. You can also see all the diffs [here](https://github.com/runtipi/runtipi-appstore/compare/master...cori:runtipi-appstore:coris-apps) (I have a couple of other unrelated apps in there)

**Reply by innuendoz** [reply]
sorry for the late response, but your suggestion worked. Thanks
